ansible/roles/clone-git-repo/tasks/main.yaml

```diff
@@ -0,0 +1,13 @@
+- name: Create dir to app
+  ansible.builtin.file:
+    path: "{{ destination }}"
+    state: directory
+    recurse: yes
+    mode: 0755
+
+- name: Clone repo from git  
+  git:
+    repo: "{{ git_repo_link }}"
+    dest: "{{ destination }}"
+    version: features-hw-24-26
```

Comment:
lets use var here to specify what branch we want to deploy

Reply:
> lets use var here to specify what branch we want to deploy

Ok, i made this

Reply:
> lets use var here to specify what branch we want to deploy

I don't quite understand what you mean